content/en/aws/opensearch/index.md

```diff
@@ -0,0 +1,415 @@
+1. Run docker compose:
+{{< command >}}
+$ docker-compose up -d
+{{< /command >}}
+
+2. Create the OpenSearch domain:
+{{< command >}}
+$ awslocal opensearch create-domain --domain-name my-cluster
+{
+    "DomainStatus": {
+        "DomainId": "000000000000/my-domain",
+        "DomainName": "my-domain",
+        "ARN": "arn:aws:es:us-east-1:000000000000:domain/my-domain",
+        "Created": true,
+        "Deleted": false,
+        "Endpoint": "my-domain.us-east-1.opensearch.localhost.localstack.cloud:4566",
+        "Processing": false,
+        "UpgradeProcessing": false,
+        "EngineVersion": "OpenSearch_1.1",
+        "ClusterConfig": {
+            "InstanceType": "m3.medium.search",
+            "InstanceCount": 1,
+            "DedicatedMasterEnabled": true,
+            "ZoneAwarenessEnabled": false,
+            "DedicatedMasterType": "m3.medium.search",
+            "DedicatedMasterCount": 1,
+            "WarmEnabled": false,
+            "ColdStorageOptions": {
+                "Enabled": false
+            }
+        },
+        "EBSOptions": {
+            "EBSEnabled": true,
+            "VolumeType": "gp2",
+            "VolumeSize": 10,
+            "Iops": 0
+        },
+        "AccessPolicies": "",
+        "SnapshotOptions": {
+            "AutomatedSnapshotStartHour": 0
+        },
+        "CognitoOptions": {
+            "Enabled": false
+        },
+        "EncryptionAtRestOptions": {
+            "Enabled": false
+        },
+        "NodeToNodeEncryptionOptions": {
+            "Enabled": false
+        },
+        "AdvancedOptions": {
+            "override_main_response_version": "false",
+            "rest.action.multi.allow_explicit_index": "true"
+        },
+        "ServiceSoftwareOptions": {
+            "CurrentVersion": "",
+            "NewVersion": "",
+            "UpdateAvailable": false,
+            "Cancellable": false,
+            "UpdateStatus": "COMPLETED",
+            "Description": "There is no software update available for this domain.",
+            "AutomatedUpdateDate": 0.0,
+            "OptionalDeployment": true
+        },
+        "DomainEndpointOptions": {
+            "EnforceHTTPS": false,
+            "TLSSecurityPolicy": "Policy-Min-TLS-1-0-2019-07",
+            "CustomEndpointEnabled": false
+        },
+        "AdvancedSecurityOptions": {
+            "Enabled": false,
+            "InternalUserDatabaseEnabled": false
+        },
+        "AutoTuneOptions": {
+            "State": "ENABLE_IN_PROGRESS"
+        }
+    }
+}
+{{< /command >}}
+
+3. If the `Processing` status is `true`, it means that the cluster is not yet healthy. You can run `decribe-domain` to receive the status:
+{{< command >}}
+$ awslocal opensearch describe-domain --domain-name my-domain
+{{< /command >}}
+
+4. Check the cluster health endpoint and create indices:
+{{< command >}}
+$ curl my-domain.us-east-1.opensearch.localhost.localstack.cloud:4566/_cluster/health | jq
+{
+  "name": "alex-notebook",
```

Comment:
Maybe use `host-pc` instead of `alex-notebook` here like further above
```suggestion
  "name": "host-pc",
```

Reply:
Thanks! Fixed.